login.php

```diff
@@ -55,11 +55,12 @@
 						<br/>
 						<li><a href="#" >Location</a> </li>
 						<select name="location" tabindex=3 class="generic" >
-							<option value="default">Default</option>
 							<?php
 								if (isset($configValues['CONFIG_LOCATIONS']) && is_array($configValues['CONFIG_LOCATIONS'])) {
```

Comment:
Nice PR. But in that case, I would add one more check to line 59:

if (isset($configValues['CONFIG_LOCATIONS']) && is_array($configValues['CONFIG_LOCATIONS']) **&& count($configValues['CONFIG_LOCATIONS']) > 0**) {

What do you think?

Reply:
Nice, it's gonna be good if the user leaves the array empty

Reply:
Added